modules/core/src/main/java/org/apache/fluo/recipes/core/map/CollisionFreeMap.java

```diff
@@ -44,12 +38,19 @@
 import org.apache.fluo.recipes.core.common.TableOptimizations.TableOptimizationsFactory;
 import org.apache.fluo.recipes.core.serialization.SimpleSerializer;
 
+import com.google.common.base.Preconditions;
```

Comment:
I'm going to write a Maven plugin to do that automatically. Please hold off on merging this until I do that... that way nobody has to manually fix the imports.